src/blog/posts/forward-to-2020.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Looking forward to 2020
+heroimage: forward-to-2020.jpg
```

Comment:
after thinking about this a little, I think it might be best to leave this implicit (shock horror coming from me i know 😀). that was it forces our images to have a certain name structure. Can just change our default file extension to jpg? I'm not going to get super hung up on it, so if you disagree, just go for it 🙂

Also, we need to update the `og:` tags in the header if we do change the image frontmatter.

Reply:
I have returned this back to implicit to save retesting social links.